lib/webvalve/manager.rb

```diff
@@ -149,6 +150,13 @@ def ensure_non_duplicate_stub(config)
       stubbed_urls << config.service_url
     end
 
+    def load_configs!
+      WebValve.config_paths.each do |root|
+        path = root.join('config', 'webvalve.rb').to_s
+        load path if File.exist?(path)
```

Comment:
Don't love that we're doing ruby file io on every test case now, but we can look to optimize if it seems to actually be slow.